specs/consensus/overview.md

```diff
@@ -1047,14 +1063,14 @@ successful round is observed.
 In other words, it should be ensured that, from `GST` on, if any correct
 process executes pseudo-code line 36 for a round `r*`, then every correct
 process executes the same line while still in round `r*`.
-This is achieved by the means of, and it is actually the reason for which
-Tendermint has, the `timeoutPrecommit(r*)`.
+This is achieved by the means of, and it is actually the key reason for which
+Tendermint has, the `timeoutPrecommit(r*)`. 
 Before moving to the next round, because the current one has not succeeded,
 processes wait for a while to ensure that they have received all the `PREVOTE`
 messages from that round, broadcast or received by other correct processes.
 As a result, if a correct process has locked or updated its valid value in
-round `r*`, then every correct process will at least update its valid value in
-round `r*`.
+round `r*`,  then every correct process will at least update its valid value in
+round `r*`, because due to gossip communication property it will receive the same set of messages. 
```

Comment:
```suggestion
round `r*`.
This happens because, due to [Gossip communication property](#network), every correct process receives the same set of messages within `∆`. 
```